packages/@azure/servicebus/data-plane/package.json

```diff
@@ -22,18 +22,18 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "@azure/amqp-common": "^1.0.0-preview.1",
+    "@azure/amqp-common": "git+https://github.com/bterlson/amqp-common-js.git#new-msrest",
+    "@azure/ms-rest-nodeauth": "^0.9.1",
     "@types/long": "^4.0.0",
     "debug": "^3.1.0",
     "is-buffer": "^2.0.3",
     "long": "^4.0.0",
-    "ms-rest-azure": "^2.5.9",
+    "rhea": "^0.3.11",
```

Comment:
Why do we need this dependency?

Reply:
I am hoping to remove this, but I had to have it there else I had a ye olde multiple-copies-of-rhea-breaking-everything-in-subtle-ways situation.